questionnaires/forms.py

```diff
@@ -86,6 +86,7 @@ def clean(self):
     def save(self, commit):
         # Tidy up the skip logic when field cant have skip logic
         for form in self.formsets[self.form_field_name]:
+            form.instance.correct_answer = form.instance.correct_answer.replace(',', u'\u201A')
```

Comment:
Surveys don't have a correct answer field, so this line should be removed.

Reply:
dont